### ISSUE TITLE
Steam crash on startup

### ISSUE THREAD

**Issue by Lunderberg** [ValveSoftware/steam-for-linux#7659]
#### Your system information

* Steam client version (build number or date): version 1613176728
* Distribution (e.g. Ubuntu): Linux Mint 20
* Opted into Steam client beta?: [Yes/No] No
* Have you checked for system updates?: [Yes/No] Yes

#### Please describe your issue in as much detail as possible:
When opening steam, the auto-update pops up and runs, but the main GUI does not appear.  Running steam from the command-line shows [the following output](https://github.com/ValveSoftware/steam-for-linux/files/5978420/steam_error_logs.txt), indicating that the steam executable segfaulted.  The crash dump is 
[attached here](https://github.com/ValveSoftware/steam-for-linux/files/5978428/dmp_file.zip).

#### Steps for reproducing this issue:

1. Start steam, either through start menu or command line
2. Wait for auto-update to finish.
3. No GUI opens, command-line messages show a segfault occurred.

**Comment by kisak-valve** [comment]
Hello @Lunderberg, swrast is mesa's slower software renderer and the fact that Steam is trying to use it hints that your system uses the nVidia proprietary driver, but don't have 32 bit userspace support installed.

Please check what video driver series you have installed, then check if `libnvidia-gl-460:i386` is installed and install it (replace 460 with the driver series you're using). The second half should be something like `apt policy libnvidia-gl-460 libnvidia-gl-460:i386` and `sudo apt install libnvidia-gl-460:i386`.

**Comment by Lunderberg** [comment]
Thank you for the fast response, and that pointed me in the right direction.  Previously, I had the 64-bit version of the nvidia driver installed, with 32-bit library support.  A few days ago, I installed CUDA, which provided a driver without 32-bit library support, which removed my earlier installed driver.

If I just installed the 32-bit driver, then I lost hardware acceleration entirely.  Instead, I needed to reinstall the previous 64-bit drives.  In case anybody else has a similar issue, [this stackexchange post](https://askubuntu.com/a/1222221) from last March was posted by somebody with the same issue.
